lib/generators/cypress_on_rails/templates/spec/playwright/support/on-rails.js

```diff
@@ -8,7 +8,7 @@ const appCommands = async (data) => {
   const response = await context.post('/__e2e__/command', { data })
 
   expect(response.ok()).toBeTruthy()
-  return response.body
+  return response.json();
 }
 
 const app = (name, options = {}) => appCommands({ name, options }).then((body) => body[0])
```

Comment:
@helio3197 Since `appCommand` now returns JSON, don't you think `then` part for `app` should be updated as well?

Reply:
I think it should remain the same since the `/__e2e__/command` endpoint returns an array of command execution results, and the current behavior is to grab the first command result as the return value as `app` handles only a command request at time